Workbooks/Cloudflare.json

```diff
@@ -0,0 +1,981 @@
+                "unit": 17,
+                "options": {
+                  "maximumSignificantDigits": 3,
+                  "maximumFractionDigits": 2
+                }
+              }
+            },
+            "showBorder": false,
+            "sortCriteriaField": "Count",
+            "sortOrderField": 2,
+            "size": "auto"
+          }
+        },
+        "conditionalVisibility": {
+          "parameterName": "tab",
+          "comparison": "isEqualTo",
+          "value": "cloudflare_security_overview"
+        },
+        "customWidth": "33",
+        "name": "Top Threat User Agents - Copy"
+      },
+      {
+        "type": 3,
+        "content": {
+          "version": "KqlItem/1.0",
+          "query": "let 5xx = Cloudflare_CL \n| where tostring(EdgeResponseStatus_d) startswith \"5\"\n| summarize Count=count()\n| extend title=\"5xx Errors (Edge)\";\n\nlet 4xx = Cloudflare_CL \n| where tostring(EdgeResponseStatus_d) startswith \"4\"\n| summarize Count=count()\n| extend title=\"4xx Errors (Edge)\";\n\nlet 3xx = Cloudflare_CL \n| where tostring(EdgeResponseStatus_d) startswith \"3\"\n| summarize Count=count()\n| extend title=\"3xx Errors (Edge)\";\n\nlet result_table = union 5xx, 4xx, 3xx; \nresult_table \n| sort by Count\n\n",
+          "size": 0,
+          "title": "ERRORS Counts (Edge)",
+          "timeContext": {
+            "durationMs": 0
+          },
+          "timeContextFromParameter": "TimeRange",
+          "queryType": 0,
+          "resourceType": "microsoft.operationalinsights/workspaces",
+          "visualization": "tiles",
+          "tileSettings": {
+            "showBorder": false,
+            "titleContent": {
+              "columnMatch": "title",
+              "formatter": 1
+            },
+            "leftContent": {
+              "columnMatch": "Count",
+              "formatter": 12,
+              "formatOptions": {
+                "palette": "auto"
+              },
+              "numberFormat": {
+                "unit": 17,
+                "options": {
+                  "maximumSignificantDigits": 3,
+                  "maximumFractionDigits": 2
+                }
+              }
+            }
+          }
+        },
+        "conditionalVisibility": {
+          "parameterName": "tab",
+          "comparison": "isEqualTo",
+          "value": "cloudflare_reliability_summary"
+        },
+        "customWidth": "33",
+        "name": "Errors (Edge)"
+      },
+      {
+        "type": 3,
+        "content": {
+          "version": "KqlItem/1.0",
+          "query": "Cloudflare_CL \n| extend response_error_type= case(tostring(EdgeResponseStatus_d) startswith \"2\" , \"2xx\", tostring(EdgeResponseStatus_d) startswith \"3\" , \"3xx\", tostring(EdgeResponseStatus_d) startswith \"4\" , \"4xx\", tostring(EdgeResponseStatus_d) startswith \"5\" , \"5xx\",\"\")\n| where isnotempty(response_error_type)\n| summarize Count=count() by response_error_type",
```

Comment:
Please replace EdgeResponseStatus_d with EdgeResponseStatus_s (the column is of string type). Please fix in all the queries with this column name.

Reply:
EdgeResponseStatus_d column is of number type.